eng/packages/http-client-csharp/.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Use unix line endings always, even on Windows
+*                            text=auto eol=lf
```

Comment:
did you mean to have the large indentation ?

Reply:
I copied from the typespec repo where I guess it did this for alignment with other lines. Still if we add other lines that use more complex patterns we will want to have indentation so I think it is fine to leave.